PubNub/Data/PNConfiguration.h

```diff
@@ -56,6 +56,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable, copy) NSString *authKey;
 
+/**
+ * @brief Token which is used along with every request to \b PubNub service to identify client user.
+ *
+ * @discussion \b PubNub service provide \b PAM (PubNub Access Manager) functionality which allow to
+ * specify access rights to access \b PubNub services with provided \c publishKey and
+ * \c subscribeKey keys.
+ * Access can be limited to concrete users. \b PAM system use this key to check whether client user
+ * has rights to access to required service or not.
+ *
+ * @warning If \c authToken is set if till be used instead of \c authKey.
+ *
+ * @default By default this value set to \b nil.
+ */
+@property (nonatomic, nullable, copy) NSString *authToken;
+
```

Comment:
As mentioned before; IMO we should use tokenManger for that

Reply:
@650elx we give user a way to specify value for `auth` during client configuration. With token also set for `auth` in query - so it is logical that we let user set it at instance initialization time.

also I mentioned above, that it is strange to use some TokenManager, which doesn’t do any management aside of single value storage.